project_name/settings/base.py

```diff
@@ -25,7 +25,7 @@
 # See https://docs.djangoproject.com/en/2.1/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.environ.get('SECRET_KEY', 'o7ql0!vuk0%rgrh9p2bihq#pege$qqlm@zo#8&t==%&za33m*2')
+SECRET_KEY = os.environ.get('SECRET_KEY', '{{ secret_key }}')
```

Comment:
Should we keep this as the literal string secret_key to ensure people remember to set a proper secret key as an environment variable?

Reply:
I think django sets a default secret key for a reason -- want apps to work in development right away. Not opposed to putting `SECRET_KEY = os.environ.get('SECRET_KEY', None)` as an override in `settings.production`, though.

Reply:
I think django generates one because it doesn't assume the secret key will be uploaded to github. For closed-source applications, that makes the most sense. Setting the default (and by proxy development) secret key to just 'secret_key' would still allow django to work in development.